global/hooks/useSingularityData/index.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ * Copyright (c) 2021 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ *  This program and the accompanying materials are made available under the terms of
+ *  the GNU Affero General Public License v3.0. You should have received a copy of the
+ *  GNU Affero General Public License along with this program.
+ *   If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ *  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ *  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import { useState } from 'react';
+
+import { getConfig } from '../../config';
+import processStream from '../../utils/processStream';
+import { ContributorsResponse, SingularityErrorResponse } from './types';
+
+const useSingularityData = () => {
+  const { NEXT_PUBLIC_MUSE_API } = getConfig();
+  const [awaitingResponse, setAwaitingResponse] = useState(false);
+
+  // For reference: https://singularity.dev.cancogen.cancercollaboratory.org/swagger-ui/
+  const fetchContributors = () => {
+    setAwaitingResponse(true);
+
+    return fetch(`${NEXT_PUBLIC_MUSE_API}contributors`, {
```

Comment:
Just a suggestion, might be better to use urlJoin then just string template so it doesn't rely on configured url to have the ending `/`